src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssProperties = sourceString
+    .trim()
```

Comment:
The use of `trim()` here is redundant since you're splitting by ';' and trimming each part individually later. You can remove this to simplify the code.